ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bfd.j2

```diff
@@ -1,8 +1,5 @@
-{% if bfd_multihop is defined and bfd_multihop is not none %}
-{%     if bfd_multihop.interval == 300 and bfd_multihop.min_rx == 300 and bfd_multihop.multiplier == 3 %}
-{%     else %}
+{% if bfd_multihop is arista.avd.defined %}
 !
 router bfd
    multihop interval {{ bfd_multihop.interval }} min-rx {{ bfd_multihop.min_rx }} multiplier {{ bfd_multihop.multiplier }}
```

Comment:
It should be validated if bfd_multihop.interval, min_rx and multiplier exist. Otherwise, command can be invalid.

Reply:
addressed with commit: d71be0c